src/index.js

```diff
@@ -28,10 +27,8 @@ import cors from 'cors';
 import ip from 'ip';
 import ProxyAgent from 'proxy-agent';
 
-const readFile = promisify(fs.readFile);
-
-const getLocalModuleMap = async ({ pathToModulemap, oneAppDevCdnAddress }) => {
-  const moduleMap = JSON.parse(await readFile(pathToModulemap, 'utf8'));
+const getLocalModuleMap = ({ pathToModulemap, oneAppDevCdnAddress }) => {
```

Comment:
i think this should still return a promise. but might not be necessary with promise-settle

Reply:
It looks like from the example on the `promise-settle` readme that you can pass non-promises as an argument:

https://www.npmjs.com/package/promise-settle